kqueen/auth/common.py

```diff
@@ -9,21 +9,66 @@
 import importlib
 import logging
 
+config = current_config()
 logger = logging.getLogger('kqueen_api')
 
+"""
+Authentication Modules
+
+To define new module, need to specify it as dictionary, where:
+
+"auth_option_lower_case": {"engine": "EqualsToAuthClassName",
+                           "param": {
+                               "key": "value"
+                           }
+                           }
+"""
+
+AUTH_MODULES = {
+    "ldap": {
+        "engine": "LDAPAuth",
+        "param": {
+            "uri": config.get('LDAP_URI')
+        }
+    },
+    "local": {
+        "engine": "LocalAuth",
+        "param": {
+
```

Comment:
This line shouldn't be blank I think, check PEP8 pls

Reply:
flake passed, let me check